.vscode/launch.json

```diff
@@ -0,0 +1,23 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Web server",
+            "type": "go",
+            "request": "attach",
+            "port": 12345,
```

Comment:
I think you will need to add `-p "12345:12345" \` to run.sh so that it will port forward 12345 from inside the container to the main host.

https://github.com/web-platform-tests/wpt.fyi/blob/594077b9c49e026214df4ad5670983260fd98082/util/docker-dev/run.sh#L99

Reply:
Done.